src/NumericalAlgorithms/LinearAlgebra/CMakeLists.txt

```diff
@@ -15,4 +15,7 @@ target_link_libraries(
   Blas
   DataStructures
   ErrorHandling
+
```

Comment:
of these three options of formatting I prefer the first two over the last tbh

```cmake
target_link_libraries(
  ${LIBRARY}
  PUBLIC
  Blas
  DataStructures
  PRIVATE
  Lapack
)
```

```cmake
target_link_libraries(
  ${LIBRARY}
  PUBLIC
    Blas
    DataStructures
  PRIVATE
    Lapack
)
```

```cmake
target_link_libraries(
  ${LIBRARY}
  PUBLIC
  Blas
  DataStructures

  PRIVATE
  Lapack
)
```

Reply:
I find the last to be the easiest by far...